config/psyclone.cfg

```diff
@@ -63,7 +63,7 @@ access_mapping = gh_read: read, gh_write: write, gh_readwrite: readwrite,
 # that it iterates over dofs. This is currently only the case for
 # builtins. If annexed dofs are computed then in certain cases we
 # remove the need for a halo exchange call.
-COMPUTE_ANNEXED_DOFS = false
+COMPUTE_ANNEXED_DOFS = true
```

Comment:
@rupertford @arporter This is needed for properly running LFRic. Should we change this default value? Or do you prefer that I sed replace it in the github workflow at let it be false by default.

Same with NUM_ANY_DISCONTINUOUS_SPACE below, LFRic uses 20.